demo/runtime/src/lib.rs

```diff
@@ -170,6 +171,42 @@ pub type Council = council::Module<Concrete>;
 /// Council voting module for this concrete runtime.
 pub type CouncilVoting = council::voting::Module<Concrete>;
 
+/// Concrete log type.
+#[derive(Clone, PartialEq, Eq, Encode, Decode)]
+#[cfg_attr(feature = "std", derive(Debug, Serialize, Deserialize))]
+pub enum ConcreteLog {
+	/// Authority changes log.
```

Comment:
more information. describe specifically under what circumstances this item is used as opposed to any other.